src/metabase/lib/util.cljc

```diff
@@ -715,19 +723,11 @@
       (when (#{:mbql/query :query :native :internal} query-type)
         query-type))))
 
-(mu/defn referenced-field-ids :- [:maybe [:set ::lib.schema.id/field]]
-  "Find all the integer field IDs in `coll`, Which can arbitrarily be anything that is part of MLv2 query schema."
-  [coll]
-  (not-empty
-   (into #{}
-         (comp cat (filter some?))
-         (lib.util.match/match coll [:field opts (id :guard int?)] [id (:source-field opts)]))))
-
-(defn collect-source-tables
```

Comment:
This was an evil function in Lib that operated on legacy MBQL. Deleted, and replaced existing usages with ones that use Lib + MBQL 5 instead.